### PR TITLE
Switch to mozlog for logging

### LINKED ISSUE #196: Use mozlog format
We should emit logs in the mozlog format.  The documentation I could find is at https://wiki.mozilla.org/Firefox/Services/Logging

### PR DESCRIPTION
**Pull request by lmorchard**
Fixes #196

### CODE REVIEW

**Comment by lmorchard** [comment]
Rebased against master after merging a few PRs

File: functions/periodicMetrics.js [hunk 1]
```diff
@@ -17,28 +24,27 @@ module.exports.handler = async function(event, context) {
...
 
   let pingCount = 0;
-  logInfo("Periodic metrics monitor start");
+  log.debug("start");
   while (context.getRemainingTimeInMillis() > pingPeriod + 1000) {
+    pingCount++;
     try {
-      await sendHeartbeatMetrics(process.env, context);
+      await sendHeartbeatMetrics(log, process.env, context);
       pingCount++;
```
Comment (chenba) [inline functions/periodicMetrics.js]: There's a new line, 29, that's also incrementing `pingCount`.  Was this line meant to be removed?

File: functions/processQueueItem.js [hunk 2]
```diff
@@ -8,22 +8,24 @@ const request = require("request-promise-native");
...
-} = require("../lib/utils.js");
-
-exports.handler = async function({ Records }, context) {
-  logInfo("Received", Records.length, "messages to process");
+const { wait, epochNow } = require("../lib/utils.js");
+
+exports.handler = async function(event = {}, context = {}) {
+  const { Records } = event;
+  const log = require("../lib/logging")({
+    name: "periodicMetrics",
```
Comment (chenba) [inline functions/processQueueItem.js]: This name doesn't look right.

File: functions/processQueueItem.js [hunk 3]
```diff
@@ -68,7 +70,17 @@ Response JSON:
 ${responseUrl}
 `;
 
-exports.handleOne = async function({ receiptHandle, body }, { awsRequestId }) {
+exports.handleOne = async function(event, context) {
+  const log = require("../lib/logging")({
+    name: "periodicMetrics.worker",
```
Comment (chenba) [inline functions/processQueueItem.js]: The prefix half of this name doesn't look right.

**Comment by lmorchard** [comment]
Rebased against master, made some tweaks per review comments

File: functions/periodicMetrics.js (hunk 1 above)
Comment (lmorchard) [reply functions/periodicMetrics.js]: Removed line 29 - count increment here in the try/catch is probably the better place

File: README.md [hunk 4]
```diff
@@ -147,8 +147,9 @@ Read this Serverless Blog post for more details: https://serverless.com/blog/ser
 
 When using `serverless deploy` to deploy the stack, you can use several environment variables to alter configuration. Note that many of these are currently overridden by a `serverless.local.yml` file, if present.
 
-- `LOG_INFO` - set to "1" for informational log output, "0" to disable (default: "1")
-- `LOG_DEBUG` - set to "1" for verbose debug log output, "0" to disable (default: "0")
+- `LOG_LEVEL` - (default: "info) one of the following severity levels for log verbosity in increasing order: critical, error, warn, info, debug, verbose, trace
```
Comment (chenba) [inline README.md]: The closing double quote for `"info` is missing.

**Pull request merged**